github.md

```diff
@@ -1,13 +1,23 @@
 ---
 order: 1000
 ---
-# GitHub Workflows
+# GitHub 
+
+## Recommended repo settings
+
+In the admin settings for repos on GitHub, the recommended settings are: 
+* default branch should be "master" (Branches > Default branch)
+* disable force push to master. For everyone. Even admins. (Branches > Protected branches)
+    * check: `protect master branch`, `require status checks`, `require that branches are up to date`, `include administrators`, `require CI to pass` if available
```

Comment:
I'm not sure what the mechanical effect of requiring status checks is in the Github UI, but given that Travis, Coveralls, etc. can be kind of hinky sometimes I would hesitate to gate a merge behind those things passing.